src/main/java/org/sefglobal/scholarx/service/IntrospectionService.java

```diff
@@ -99,19 +99,25 @@ public Profile getLoggedInUser(long id)
      * Retrieves all the {@link Program} objects where the user is a {@link Mentor}
```

Comment:
Description is inaccurate due to the new filter

Reply:
I'll update the description, thanks